js/apps/system/_admin/aardvark/APP/frontend/js/collections/arangoCollections.js

```diff
@@ -167,7 +167,11 @@
         data.shardKeys = object.shardBy;
       }
       if (object.replicationFactor) {
-        data.replicationFactor = JSON.parse(object.replicationFactor);
+        if (typeof object.replicationFactor === 'number' && frontendConfig.isEnterprise) {
+          data.replicationFactor = JSON.parse(object.replicationFactor);
+        } else {
+          data.replicationFactor = object.replicationFactor;
+        }
       }
```

Comment:
@hkernbach : if something is a number already, it does not need to be parsed to become a number. 
Is something like this intended?
```js
      if (object.replicationFactor) {
        data.replicationFactor = object.replicationFactor;
        if (object.replicationFactor.test(/^[0-9]+$/)) {
          // looks like a number
          data.replicationFactor = JSON.parse(object.replicationFactor);
        } 
      }
```